Add MultipleScalersCalculation explanation (#505)
kedacore/keda-docs: content/docs/2.5/concepts/scaling-jobs.md

      - "Ready"
      - "PodScheduled"
      - "AnyOtherCustomPodCondition"

>     MultipleScalersOption: "max" # Optional. Default: max. If multiple scaler is exists, how to calculate the queueLength and maxValue

Reviewer: ```suggestion
    MultipleScalersOption: "max" # Optional. Default: max. Specifies how to calculate the queueLength and maxValue when multiple scaler are used.
```

>     MultipleScalersOption: "max" # Optional. Default: max. Specifies how to calculate the queueLength and maxValue when multiple scaler are used.

Reviewer: ```suggestion
    multipleScalersOption: "max" # Optional. Default: max. Specifies how to calculate the queueLength and maxValue when multiple scaler are used.
```

Reviewer 2: What about naming this property `multipleScalersCalculation` or something like that?

Author: Sure. I'll change the code.

>     multipleScalersCalculation : "max" # Optional. Default: max. Specifies how to calculate the queueLength and maxValue when multiple scaler are used.

Reviewer 2: ```suggestion
    multipleScalersCalculation : "max" # Optional. Default: max. Specifies how to calculate the target metrics when multiple scalers are defined.
```

```
For more details,  you can refer to [this PR](https://github.com/kedacore/keda/pull/1227).

---

```yaml
scalingStrategy:

>     MultipleScalersOption: "max" # Optional. Default: max. If multiple scaler is exists, how to calculate the queueLength and maxValue

Reviewer: ```suggestion
    multipleScalersOption: "max" # Optional. Default: max. If multiple scaler is exists, how to calculate the queueLength and maxValue
```

>     multipleScalersCalculation : "max" # Optional. Default: max. If multiple scaler is exists, how to calculate the queueLength and maxValue

Reviewer 2: ```suggestion
    multipleScalersCalculation : "max" # Optional. Default: max. Specifies how to calculate the target metrics (`queueLength` and `maxValue`) when multiple scalers are defined.
```

```
Select a behavior if you have multiple triggers. Possible values are `max`, `min`, `avg`, or `sum`. The default value is `max`. 

> * **max:** default, pick a scaler that has the max number of `queueLength`.

Reviewer: ```suggestion
* **max:** - Use the scaler that has the max number of `queueLength`. (default)
```

> * **max:** default, pick a scaler that has the max number of `queueLength`.

Reviewer: What does queueLength refer to here? Is that a metric that I have defined or an internal one? Do end-users care about this metric or should it be "pending jobs"?

Author: Thank you, @tomkerkhove for the great review! 

queueLength is the metadata that is included on scaler.GetMetrics. It is the number of the queue that is not consumed. In case of "pending jobs" it might confusing since ScaledJob create jobs and some of them called pending jobs.  I'm wondering what is the proper expression in English. 

https://github.com/kedacore/keda/blob/main/pkg/scaling/scale_handler.go#L283
For the DCO issue that is created GitHub gui, I'll fix it. later. :)

Author: Hi @tomkerkhove If you don't have an idea, how about adding explanation of the `queueLength` ? e.g. "queueLength is the number of the queues that is not consumed. The value is returned from each scalers"

Reviewer: It's ok for me

> * **min:** pick a scaler that has the min number of `queueLength`.

Reviewer: ```suggestion
* **min:** - Use the scaler that has the min number of `queueLength`.
```

> * **avg:** sum up all the active scalers metrics and devided by the number of the active scalers.

Reviewer: ```suggestion
* **avg:** - Sum up all the active scalers metrics and divide by the number of active scalers.
```

> * **sum:** sum up all the active scalers metrics.

Reviewer: ```suggestion
* **sum:** - Sum up all the active scalers metrics.
```

> * **max:** - Use the scaler that has the max number of `queueLength`. (default)
> * **min:** - Use the scaler that has the min number of `queueLength`.
> * **avg:** - Sum up all the active scalers metrics and divide by the number of active scalers.
> * **sum:** - Sum up all the active scalers metrics.

Reviewer 2: ```suggestion
* **max:** - Use metrics from the scaler that has the max number of `queueLength`. (default)
* **min:** - Use metrics from the scaler that has the min number of `queueLength`.
* **avg:** - Sum up all the active scalers metrics and divide by the number of active scalers.
* **sum:** - Sum up all the active scalers metrics.
```
